src/wp-includes/deprecated.php

```diff
@@ -6239,3 +6239,60 @@ function the_block_template_skip_link() {
 	</script>
 	<?php
 }
+
+/**
+ * Ensure that the view script has the `wp-interactivity` dependency.
+ *
+ * @since 6.4.0
+ * @deprecated 6.5.0
+ *
+ * @global WP_Scripts $wp_scripts
+ */
+function block_core_query_ensure_interactivity_dependency() {
+	_deprecated_function( __FUNCTION__, '6.5.0', 'Used wp_register_script_module instead' );
```

Comment:
```suggestion
	_deprecated_function( __FUNCTION__, '6.5.0', 'wp_register_script_module' );
```

Reply:
Done in https://github.com/WordPress/wordpress-develop/pull/5973/commits/b6af53403e501bb2dc8d7083a16d657bc1c53b55